packages/design-system/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as TooltipComponent } from "./Tooltip";
```

Comment:
I think `export * from './Tooltip` is sufficient if you can change 
`export default TooltipComponent;` to `export TooltipComponent;`;

Reply:
Will take this up in separate PR